### PR TITLE
2.0.0 Testnet documentation

### PR DESCRIPTION
**Pull request by badrogger**

### CODE REVIEW

File: validators/fuji/fuji-versions.md [hunk 1]
```diff
@@ -33,39 +34,39 @@ If you're using `ufw` or `iptables` for configuring your own rules, please, do n
...
 TestNet Environment Variables: 
 
 ```shell
 MONITORING_CONTAINERS=True
-DOCKER_LVMPY_STREAM=1.0.1-beta.2
-MANAGER_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/rinkeby/skale-manager/1.8.0/skale-manager-1.8.0-beta.1-rinkeby-abi.json
-IMA_CONTRACTS_ABI_URL=https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/rinkeby/IMA/1.0.0-beta.2/abi.json
-CONTAINER_CONFIGS_STREAM=1.7.3-testnet
+DOCKER_LVMPY_STREAM=1.0.1-beta.3
+MANAGER_CONTRACTS_ABI_URL=https://github.com/skalenetwork/skale-network/blob/master/releases/rinkeby/skale-manager/1.8.1/skale-manager-1.8.1-beta.1-rinkeby-abi.json
```
Comment (dmytrotkk) [inline validators/fuji/fuji-versions.md]: Wrong link, change to raw

File: validators/fuji/soft-upgrade-testnet.md [hunk 2]
```diff
@@ -57,28 +70,23 @@ cd  ~ && vi .env
...
-CONTAINER_CONFIGS_STREAM=1.7.3-testnet
-FILEBEAT_HOST=filebeat.testnet.skalenodes.com:5001
-DISABLE_IMA=True
+DOCKER_LVMPY_STREAM=1.0.1-beta.3
+CONTAINER_CONFIGS_STREAM=2.0.0-testnet.0
+DISABLE_IMA=False
 ENV_TYPE=testnet
+MONITORING_CONTAINERS=True
+DOCKER_LVMPY_STREAM=1.0.1-beta.3
+MANAGER_CONTRACTS_ABI_URL=https://github.com/skalenetwork/skale-network/blob/master/releases/rinkeby/skale-manager/1.8.1/skale-manager-1.8.1-beta.1-rinkeby-abi.json
```
Comment (dmytrotkk) [inline validators/fuji/soft-upgrade-testnet.md]: same here

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: validators/fuji/soft-upgrade-testnet.md (hunk 2 above)
Comment (badrogger) [reply validators/fuji/soft-upgrade-testnet.md]: addressed

File: validators/fuji/fuji-versions.md (hunk 1 above)
Comment (badrogger) [reply validators/fuji/fuji-versions.md]: addressed

**Comment by ebruengwall-ant** [comment]
I didn't check the version numbers but the documentation looks good.

**Comment by ebruengwall-ant** [comment]
Please review and push this tomorrow for Fabio

**Review by ebruengwall-ant — APPROVED** [review]

**Review by cstrangedk — APPROVED** [review]

**Pull request merged**